RELNOTES

```diff
@@ -1,5 +1,10 @@
 firejail (0.9.68rc2) baseline; urgency=low
   * work in progress
+  * security: the firejail package on Ubuntu was left vulnerable to
+    CVE-2021-26910 for months after a patch for it was made available; the PPA
+    is now recommended on README.md instead (#4748)
```

Comment:
I don't understand the `on README.md` in this sentence. Did you add this by accident?

Reply:
@reinerh [on Feb 1](https://github.com/netblue30/firejail/pull/4889#discussion_r797061501):

> I don't understand the `on README.md` in this sentence. Did you add this by
> accident?

I tried to make the entry look consistent by itself and also look similar to
what the PR says.  But indeed it looks better without that part; removed.